pkg/agent/templates/templates.go

```diff
@@ -19,6 +19,11 @@ path = "{{ .NodeConfig.Containerd.Opt }}"
 [plugins.cri]
 stream_server_address = "{{ .NodeConfig.AgentConfig.NodeName }}"
 stream_server_port = "10010"
+{{ if eq .NodeConfig.AgentConfig.PauseImage "" }}
+sandbox_image = "k8s.gcr.io/pause:3.1"
```

Comment:
Thanks for updating @leo1971 !

Instead of setting a default here I think we should probably let containerd use its own default, so set sandbox_image only if PauseImage is not equal to "".

Also, we usually end up squashing multiple commits to a single commit on merge, so if you would like to squash or fix them up yourself feel free. Thanks! :)